src/material/datepicker/datepicker-base.ts

```diff
@@ -232,10 +232,26 @@ export interface MatDatepickerControl<D> {
   stateChanges: Observable<void>;
 }
 
+/** A picker panel that can be associated with a {@link MatDatepickerControl}. */
+export interface MatDatepickerPanel<C extends MatDatepickerControl<D>, S,
```

Comment:
Should this interface be implemented by `MatDatepicker` and `MatDateRangePicker`?

Reply:
Both extend `MatDatepickerBase`, which already implements this interface.